### DISCUSSION TITLE
Feature request: Temperature setting for ironing

### DISCUSSION THREAD

**Discussion by RobXXR** [OrcaSlicer/OrcaSlicer#2055] [Ideas]
I like to run my BBL X1-Carbon fast
I have the 3rd party nozzle/hotend with 3 holes for better heat transfer to the filament.
So I run a 0.6 mm nozzle, 0.4 mm layers, print temperature @ 135 C for PLA, which is the maximum the heater can provide
It goes fine, but when I add ironing to my print too, I always get heat creep and the filament gets stuck so, I have to take the extruder and nozzle apart to clean out the stuck filament.
Would it be possible to add a temperature setting, just for ironing?

**Comment by Killrsheep** [comment]
Not only that, for certain materials like matte and silk I've noticed that i can get good layer adhesion with some temps BUT if im ironing the print, the top ironed layer will look great if i manually reduce the temperature to another setting while its ironing

**Reply by TheDIYGuy999** [reply]
I think this is a good idea, because Ironing tends to clogging on Bambu Lab printers. Another helpful option could be lifting the nozzle ab it for ironing, let's say 0.05mm and then increasing the flow a bit instead

**Reply by BlueByLiquid** [reply]
I would like to add so many settings to ironing including the temperature settings. I'm happy to do it myself but new to orca slicer dev and not sure where to start and no one responded to even point me in the right direction for where ironing is handled in Orca and I don't have time to fully do a deep dive in getting started on Orca dev and can't find any resources.

**Comment by ValkyrieDomi** [comment]
Upvote! I like some PETG to iron hot, gives nice shiny finish.

**Comment by xTremePower** [comment]
I just tested ironing on my DIY 1m cube printer, because a customer want a big plate like part in PETG.
If I manually add temp change in the gcode to increase temp, the results are far better

PS : I print it in 1.0mm nozzle with 0.5mm /layer with ironing at 0.4mm spacing and 60mmsec speed

But every time I re slice the file after modifications, I ned to manually add the temp change lines ...

PS : you can also add a function in ironing to stop the Cooling fan  (or at least set the sped of the fan) because it's ultra shiny without cooling on ironing

**Comment by glococo** [comment]
I also found that ironing PLA at 15º more looks much better (in my case mate black)
Hope this feat arrive some day.

**Comment by CSV-12** [comment]
Great idea.

**Comment by Makemo07** [comment]
Would like to push this idea. The ideal temperatures for layer/interline adhesion and surface quality are often different then for good ironing surfaces (10-20° lower) especially for matte filament.

Would it be difficult to integrate in the advanced settings under quality>ironing or the filament settings?
